src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,44 @@
+.stars--4 {
+  display: flex;
+  padding-top: 16px;
+
+  & .stars__star:nth-child(1) {
+    display: block;
+    background-image: url(../images/star-active.svg);
+    width: 13px;
+    height: 12.2px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), all values in the CSS should be multiples of 4. The width here is 13px, which is not a multiple of 4.